app-ios/Sources/SearchFeature/SearchView.swift

```diff
@@ -50,11 +51,15 @@ public struct SearchView: View {
                     store.send(.view(.searchWordChanged($0)))
                 }
             ),
+            isPresented: $isfocused,
             placement: .navigationBarDrawer(displayMode: .always)
         )
         .foregroundStyle(AssetColors.Surface.onSurface.swiftUIColor)
         .onAppear {
             store.send(.view(.onAppear))
+            DispatchQueue.main.async {
+                isfocused = true
+            }
```

Comment:
SwiftUI.View is isolated on MainActor, so I think run this code on main thread isn't needed 👀 
How about following?

```suggestion
            isfocused = true
```

Reply:
@ry-itto 
Thank you for comment.
I have added a delay using `DispatchQueue` after reading this article:

https://www.rk-k.com/archives/7925

If the issue is resolved with the latest SwiftUI.View as mentioned in the comments, I am considering changing to the suggested approach.

Reply:
Doesn't it actually work without Dispatch?👀
If so, I think there should be comment for this issue.

Reply:
Thank you for your feedback. After testing on the simulator, everything worked without any issues. Therefore, I have removed the Dispatch.